source/codegen/service_helpers.py

```diff
@@ -4,22 +4,23 @@ def get_include_guard_name(config, suffix):
     include_guard_name = config['namespace_component'] + "_grpc" + suffix
     return include_guard_name.upper()
 
-def is_string_arg(parameter):
-    return parameter['type'] in ['ViChar[]', 'ViInt8[]', 'ViUInt8[]']
-
 def create_args(parameters):
     result = ''
     is_twist_mechanism = common_helpers.has_ivi_dance_with_a_twist_param(parameters)
-    if is_twist_mechanism:
+    if is_twist_mechanism :
         twist_value = common_helpers.get_twist_value(parameters)
         twist_value_name = common_helpers.camel_to_snake(twist_value)
     for parameter in parameters:
       parameter_name = common_helpers.camel_to_snake(parameter['cppName'])
       is_array = common_helpers.is_array(parameter['type'])
       is_output = common_helpers.is_output_parameter(parameter)
-      if is_output and is_string_arg(parameter):
+      if is_output and common_helpers.is_string_arg(parameter):
         type_without_brackets = common_helpers.get_underlying_type_name(parameter['type'])
-        result = f'{result}({type_without_brackets}*){parameter_name}.data(), '
+        if common_helpers.is_enum(parameter):
+          arg = f'response->mutable_{parameter_name}_raw()->data()'
```

Comment:
Looking at other similar usages, should the last part be `->mutable_data()` instead of `->data()`?

Reply:
I am using the method used in the normal ViUInt8 arrays as that wont require change in this code